packages/fxa-shared/subscriptions/types.ts

```diff
@@ -23,13 +26,16 @@ export interface PlanMetadata {
 
 // https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=COPS&title=SP+Tiered+Product+Support#SPTieredProductSupport-MetadataAgreements
```

Comment:
Might be worth updating this URL to point at [the ecosystem docs](https://mozilla.github.io/ecosystem-platform/docs/fxa-engineering/fxa-sub-platform#product-metadata) while you're in here?

Reply:
It should definitely be updated, I'm not sure when these options were added and how they are applied though.